docs/source/install/profile.rst

```diff
@@ -41,6 +41,12 @@ Queue: gpu (4x NVIDIA P100 16GB)
 .. literalinclude:: profiles/hemera-hzdr/gpu_picongpu.profile.example
    :language: bash
 
+Queue: fwkt_v100 (4x NVIDIA V100 32GB)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: profiles/hemera-hzdr/gv100_picongpu.profile.example
```

Comment:
please update the file name to: `profiles/hemera-hzdr/fwkt_v100_picongpu.profile.example`